src/main/java/com/splunk/logtosplunk/loggable_events/LoggableBlockEvent.java

```diff
@@ -0,0 +1,146 @@
+package com.splunk.logtosplunk.loggable_events;
+
+import com.splunk.logtosplunk.actions.BlockEventAction;
```

Comment:
This makes sense.  I think combining in the enums in with their respective classes is a good idea. Also player movement can reasonably be called a player event -we can always break the classes apart again if we want to.